consensus/utils.go

```diff
@@ -20,19 +20,53 @@ func IsValidSuperMajorityQuorumCertificate(qc QuorumCertificate, validators []Va
 	// Compute the signature that validators in the QC would have signed
 	signaturePayload := GetVoteSignaturePayload(qc.GetView(), qc.GetBlockHash())
 
+	isValidSuperMajority, validatorPublicKeysInQC := checkSuperMajorityStakeForAggregatedSignature(qc.GetAggregatedSignature(), validators)
```

Comment:
nit: the result is always valid. The bool just tells us whether or not there is a supermajority
```suggestion
	hasSuperMajorityStake, signerPublicKeys := checkSuperMajorityStakeForAggregatedSignature(qc.GetAggregatedSignature(), validators)
```